apps/src/code-studio/pd/workshop_dashboard/local_summer_workshop_survey_results.jsx

```diff
@@ -102,6 +116,43 @@ const LocalSummerWorkshopSurveyResults = React.createClass({
     );
   },
 
+  renderRow(row, i) {
+    let scoreCells;
+    let thisWorkshopData = this.state.thisWorkshop[row['key']];
+
+    if (this.state.facilitatorBreakdown && typeof thisWorkshopData === 'object') {
+      scoreCells = this.state.facilitatorNames.map((facilitator, i) => {
+        return (
+          <td key={i}>
+            {this.renderScore(row, thisWorkshopData[facilitator])}
+          </td>
+        );
+      });
```

Comment:
While using the same variable, `i`, here in the arrow function and the outer method is supported in ES6 scoping, I'd prefer a different variable name for clarity. You could even remove `i` from the `map` function and use something like `facilitator.id` as the key.